examples/demo-advanced-tables-images-upload.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<p>I think this got left - thanks for removing code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Caltech-IPAC/firefly_client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>